packages/sdk/src/core/defaultTediousConnectionConfiguration.ts

```diff
@@ -121,9 +123,13 @@ export class DefaultTediousConnectionConfiguration {
    * @returns Tedious connection configuration with username and password.
    * @internal
    */
-  private generateDefaultConfig(sqlConfig: SqlConfiguration): ConnectionConfig {
+  private generateDefaultConfig(
+    sqlConfig: SqlConfiguration,
+    databaseName?: string
+  ): ConnectionConfig {
+    const dbName = databaseName ?? sqlConfig.sqlDatabaseName;
```

Comment:
What the expected behaviour when databaseName is empty? 

Be careful that `""??"test"` will result `""`

Reply:
Because the developer explicitly pass the parameter, the empty string "" will be used in tedious config to connect database and depends on tedious behavior (probably runtime exception)

Reply:
Yes, I added warning message.